packages/auth/src/signature/middleware.test.ts

```diff
@@ -405,55 +405,6 @@ describe('Signature Service', (): void => {
       scope.isDone()
     })
 
-    test('middleware succeeds if BYPASS_SIGNATURE_VALIDATION is true with bad signature', async (): Promise<void> => {
-      const altDeps = await initIocContainer({
-        ...Config,
-        bypassSignatureValidation: true
-      })
-
-      const altContainer = await createTestApp(altDeps)
-      appContainers.push(altContainer)
```

Comment:
You could also revert `appContainers` to `appContainer`
https://github.com/interledger/rafiki/pull/683/files#diff-9c6c5d794c9dfa6afb4129ccc2ffb701bcdecb70308c63c7e4d615e8d903859c